Project.toml

```diff
@@ -21,11 +21,11 @@ TensorBoardLogger = "899adc3e-224a-11e9-021f-63837185c80f"
 [compat]
 Arrow = "2.3"
 ArrowTypes = "1, 2"
-CairoMakie = "0.7, 0.8"
+CairoMakie = "0.11"
 DataFrames = "1.3"
 Legolas = "0.5"
-Makie = "0.17.4, 0.18"
-StatsBase = "0.33"
+Makie = "0.19, 0.20"
```

Comment:
should this be only 0.20 if we use the new syntax?

Reply:
```suggestion
Makie = "0.20"
```

good catch